sdks/java/core/src/main/java/org/apache/beam/sdk/coders/AvroCoder.java

```diff
@@ -356,6 +358,10 @@ protected AvroCoder(Class<T> type, Schema schema, boolean useReflectApi) {
     return type;
   }
 
+  public boolean usesReflection() {
```

Comment:
Is this fine with the convention or should if be named `isUseReflection`

Reply:
I think we should use only one verb - either `is` or `use`